packages/frontend-plugin-api/src/components/ErrorBoundary.tsx

```diff
@@ -14,35 +14,14 @@
  * limitations under the License.
  */
 
-import React, { Component, PropsWithChildren } from 'react';
-// TODO: Dependency on MUI should be removed from core packages
-import { Button } from '@material-ui/core';
-import { ErrorPanel } from '@backstage/core-components';
+import React, { Component, ComponentType, PropsWithChildren } from 'react';
 import { BackstagePlugin } from '../wiring';
+import { CoreErrorBoundaryFallbackProps } from '../types';
 
-type DefaultErrorBoundaryFallbackProps = PropsWithChildren<{
+type ErrorBoundaryProps = PropsWithChildren<{
   plugin?: BackstagePlugin;
-  error: Error;
-  resetError: () => void;
+  fallback: ComponentType<CoreErrorBoundaryFallbackProps>;
```

Comment:
Let's uppercase this just to make it clear that it's a component. Happy to merge and do that in a followup though if you wanna merge while the build is green, since it's internal anyway.

Reply:
Sure, I'll merge and create a followup 🙌🏻

Reply:
https://github.com/backstage/backstage/pull/21737